Created the sweep_from function

ANUSHMANGAL01 opened the pull request:
This function assumes that the input tile which it receives does not have a flag, nor a question mark. It receives the tile inputted by the user, if it has a mine, it makes the loss variable true.
If the input tile has a number other than 0, it simply opens up that tile. 
If the input tile has 0, it flood fills all the zeros and opens up other tiles accordingly. It has to check individually for the corner tiles and the boundary tiles as they have different neighbours.
If any tile which was flagged or had question mark, but got opened during flood fill, then the function changes the status of is_flagged and is_q_marked arrays. 

The function also updates the value of num_swept.

ANUSHMANGAL01 committed:
Created the sweep_from function

This function assumes that the input tile which it receives does not have a flag, nor a question mark. It receives the tile inputted by the user, if it has a mine, it makes the loss variable true.
If the input tile has a number other than 0, it simply opens up that tile. 
If the input tile has 0, it flood fills all the zeros and opens up other tiles accordingly. If any tile which was flagged or had question mark, but got opened during flood fill, then the function changes the status of is_flagged and is_q_marked arrays. The function also updates the value of num_swept.

MathewKJ2048 commented:
1)Please replace the int to char conversion array using this technique:
    int n;
    char ch;
    ch = (char)(48+n);
    n = (int)(ch-48)
since the ASCII code for '0' is 48 and '1' is 49 and so forth

MathewKJ2048 commented:
Also, now that skeleton has been committed, make all future pull requests using code written inside skeleton. The auto merging might run into problems otherwise.

2) try using -10 as a placeholder instead of 10. That way, if errors occur during testing, it will be easier to spot

ANUSHMANGAL01 committed:
Add files via upload

ANUSHMANGAL01 committed:
Changed visited zero's number from 10 to -10.

I also removed the ch array and used 
state[i][j]=(char)(48+number[i][j]);

instead.

ANUSHMANGAL01 committed:
Merge pull request #12 from MathewKJ2048/main

Taking in class.txt into sweep_from branch

ANUSHMANGAL01 committed:
Added the sweep_from function and also a utility function sweep_util which will be called by sweep_from()
